src/plugins/index_pattern_management/public/components/edit_index_pattern/indexed_fields_table/components/table/table.tsx

```diff
@@ -185,6 +185,13 @@ export class Table extends PureComponent<IndexedFieldProps> {
             />
           </span>
         ) : null}
+        {field.displayName !== field.name && (
+          <div>
+            <EuiBadge iconType="eye" iconSide="left">
+              {field.displayName}
+            </EuiBadge>
+          </div>
+        )}
```

Comment:
FYI: @andreadelrio , I've added the displayName as a `ÈuiBadge`, do you think this is fine? Or is there a better option, should be a minimal change, wouldn't add a column for this prop (Maybe later on the index pattern field management would be a good use case for `EuiDataGrid`)
<img width="1118" alt="Bildschirmfoto 2020-08-25 um 10 10 55" src="https://user-images.githubusercontent.com/463851/91151019-68391400-e6bd-11ea-96be-5fff362de9c7.png">

Reply:
@kertal I think the badge works. We usually use the eye icon to represent visibility (toggling visibility to be exact) so best to reserve that icon for that use. While there's not a specific icon that I think would be a perfect fit for this use case I think it makes more sense to use the `flag` icon. I'd also consider adding a tooltip so when the use hovers over the badge it reads something like "This field has a custom label" (or display name) not sure which one you're going with? I saw screenshots with "custom label" but "display name" showed up locally.

Also, I think the form could benefit from some copy letting the user know they can customize the label/display name of the field. This is just a placeholder that we can improve with the help of the writers.
<img width="578" alt="image 45" src="https://user-images.githubusercontent.com/4016496/91221007-e4d5ed80-e6d1-11ea-941e-e808a56255e0.png">

Reply:
> @kertal I think the badge works. We usually use the eye icon to represent visibility (toggling visibility to be exact) so best to reserve that icon for that use. While there's not a specific icon that I think would be a perfect fit for this use case I think it makes more sense to use the `flag` icon. I'd also consider adding a tooltip so when the use hovers over the badge it reads something like "This field has a custom label" (or display name) not sure which one you're going with? I saw screenshots with "custom label" but "display name" showed up locally.
> 
> Also, I think the form could benefit from some copy letting the user know they can customize the label/display name of the field. This is just a placeholder that we can improve with the help of the writers.
> <img alt="image 45" width="578" src="https://user-images.githubusercontent.com/4016496/91221007-e4d5ed80-e6d1-11ea-941e-e808a56255e0.png">

I've changed icon, added tooltip, and added additional content. Unless there is a good reason against is, I think `Custom Label` is the better user facing name for the field (`displayName` is can be the customLabel, the name of the field, or a modified version of it, so it's the internal name for what is actually displayed)